PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,3 +24,14 @@ branch | PR
 ------ | ------
 fix/other_pr | [link]()
 feature/other_pr | [link]()
+
+## How to test
+Include any information that may be helpful to the reviewer(s).
+_This might include:_
+
+- links to sample pages to test
+- Any local environmental setup that is unusual (env vars, API version to point to, etc)
```

Comment:
Can we say here "...setup, **including the** unusual (env vars..."

Reply:
Thanks @JonellaCulmer, I made the change to each of the PR templates for openfec, fec-cms, and fec-eregs